tests/keras/backend/mxnet_sparse_test.py

```diff
@@ -160,6 +160,5 @@ def test_sparse_concat_axis_non_zero(self):
         assert k_s_d.shape == k_d.shape
         assert_allclose(k_s_d, k_d, atol=1e-05)
 
-
```

Comment:
Please add sparse test here, add a case where sparse_grad is true:  https://github.com/awslabs/keras-apache-mxnet/blob/master/tests/keras/layers/embeddings_test.py